serverless/pages/search-your-data.asciidoc

```diff
@@ -1,26 +1,33 @@
 [[elasticsearch-search-your-data]]
 = Search your data
 
-// :description: Use the search API to run queries on your data.
+// :description: Learn options for searching your data in Elasticsearch Serverless.
 // :keywords: serverless, elasticsearch, search
 
-A search query, or query, is a request for information about data in {es} data streams or indices.
+Searching your data in {es-serverless} works the same way as in other Elasticsearch deployments.
+If you haven't used {es} before, you can learn the basics in the {ref}/elasticsearch-intro.html[core {es} documentation].
 
-You can think of a query as a question, written in a way {es} understands. Depending on your data, you can use a query to get answers to questions like:
+You can use the https://www.elastic.co/docs/api/doc/elasticsearch-serverless[{es-serverless} REST APIs] to search your data using any HTTP client, including the <<elasticsearch-clients,{es} client libraries>>, or directly in the Dev Tools <<devtools-run-api-requests-in-the-console, Console>>.
```

Comment:
`Dev Tools` is not in the title of this product. target refers to it simply as `Console` so we might want to match that here.

```suggestion
You can use the https://www.elastic.co/docs/api/doc/elasticsearch-serverless[{es-serverless} REST APIs] to search your data using any HTTP client, including the <<elasticsearch-clients,{es} client libraries>>, or directly in <<devtools-run-api-requests-in-the-console, Console>>.
```